content-src/components/ActivityFeed/ActivityFeed.js

```diff
@@ -196,7 +196,7 @@ const GroupedActivityFeed = React.createClass({
               {sites.map((site, i) => {
                 globalCount++;
                 return (<ActivityFeedItem
-                    key={site.cacheKey || i}
+                    key={i}
```

Comment:
Yeah we basically just need unique ids... @rittme is going to take care of it for us 👍